dolweb/downloads/templates/downloads-index.html

```diff
@@ -31,10 +31,13 @@
 <h1>{% trans "Development versions" %}</h1>
 
 <div class="alert">{% blocktrans %}
-    Development versions are released every time a developer makes a change to
+    <p>Development versions are released every time a developer makes a change to
     Dolphin, several times every day! Using development versions enables you to
     use the latest and greatest improvements to the project. They are however
-    less tested than stable versions of the emulator.
+    less tested than stable versions of the emulator.</p>
+
+    <p>The development versions require 64 bit Visual C++ runtimes for Visual Studio 2015
+    installed: <a href="https://www.microsoft.com/en-us/download/details.aspx?id=48145">Link to download</a></p>
```

Comment:
I would make the `64-bit Visual C++ runtimes` part the link text, so it can run in sentence form.

e.g.
The development builds require the [64-bit Visual C++ runtimes](somelink) for Visual Studio 2015 to be installed.